app/src/main/java/org/fossasia/openevent/app/module/event/list/EventListFragment.java

```diff
@@ -136,7 +164,7 @@ public void onStop() {
 
     private void setupRecyclerView() {
         if (!initialized) {
-            eventListAdapter = new EventsListAdapter(getPresenter().getEvents(), bus);
+            eventListAdapter = new EventsListAdapter(getPresenter().getEvents(SORTBYDATE), bus);
```

Comment:
Let it be `getEvents`

Reply:
But getEvents() expects an integer argument

Reply:
Remove that. Create an explicit sort function for sorting, getEvent should remain as is